Source/WebCore/platform/network/curl/CurlRequest.h

```diff
@@ -150,6 +150,8 @@ class CurlRequest : public ThreadSafeRefCounted<CurlRequest>, public CurlRequest
     void updateHandlePauseState(bool);
     bool isHandlePaused() const;
 
+    bool shouldAbortConnectRequest(const URL&, long, long);
```

Comment:
Please add argument names `statusCode` and `httpConnectCode` here.

Reply:
But, I don't think a new member function should be added for this.
A local variable or local lambda function seem enough.

bool shouldAbortConnectRequest = ...;
if (shouldAbortConnectRequest)

auto shouldAbortConnectRequest = [] { return ... };
if (shouldAbortConnectRequest())

WDYT?

Reply:
> bool shouldAbortConnectRequest = ...;
> if (shouldAbortConnectRequest)

Yeah. This is simpler and better.